.github/workflows/e2e-soak-trigger.yaml

```diff
@@ -0,0 +1,51 @@
+name: E2ESoakTrigger
+on:
+  schedule:
+    - cron: '0 */3 * * *'
+jobs:
+  resolve_cluster:
+    permissions:
+      id-token: write # aws-actions/configure-aws-credentials@v4.0.1
+    # rmif: github.repository == 'aws/karpenter-provider-aws'
+    runs-on: ubuntu-latest
+    outputs:
+      PREEXISTING_CLUSTERS: ${{ steps.list_clusters.outputs.PREEXISTING_CLUSTERS }}
+    steps:
+    - uses: actions/checkout@v4
+    - name: configure aws credentials
+      uses: aws-actions/configure-aws-credentials@010d0da01d0b5a38af31e9c3470dbfdabdecca3a
+      with:
+       role-to-assume: arn:aws:iam::${{ vars.ACCOUNT_ID }}:role/${{ vars.ROLE_NAME }}
+       aws-region: eu-north-1
+    - uses: actions/setup-go@v4
```

Comment:
We're not pinning our actions to SHAs and we should be here

Reply:
I think that there's also a miss on the `ResourceCount` workflow on this

Reply:
Checked `ResourceCount`, and seems to be updated